seqr/utils/file_utils_tests.py

```diff
@@ -30,34 +30,3 @@ def test_mv_file_to_gs(self, mock_logger, mock_subproc):
         mock_subproc.Popen.assert_called_with('gsutil mv /temp_path gs://bucket/target_path', stdout=mock_subproc.PIPE, stderr=mock_subproc.STDOUT, shell=True)
         mock_logger.info.assert_called_with('==> gsutil mv /temp_path gs://bucket/target_path', None)
         process.wait.assert_called_with()
-
-    @mock.patch('seqr.utils.file_utils.subprocess')
-    @mock.patch('seqr.utils.file_utils.logger')
-    def test_get_gs_file_list(self, mock_logger, mock_subproc):
```

Comment:
Remove the utility function test since it doesn't increase coverage.

Reply:
I'm updating the test cases for the `get_anvil_vcf_list` and `validate_anvil_vcf` APIs. I can mock the subprocess.wait.return_value to 1 and test the error case.